Jenkinsfile

```diff
@@ -385,6 +385,60 @@ pipeline {
                         }
                     }
                 }
+             stage('cart-master-nightly') {
+                when { branch 'master' }
+                agent {
+                    label 'ci_vm1'
+                }
+                options {
+                    timeout(time: 60, unit: 'MINUTES')
+                }
+                steps {
+                    provisionNodes NODELIST: env.NODELIST,
+                        node_count: 1,
+                        snapshot: true
+                    runTest stashes: [ 'CentOS-install', 'CentOS-build-vars' ],
+                        script: """set -x
+                            . ./.build_vars.sh
+                            IOF_BASE=\${SL_PREFIX%/install*}
+                            NODELIST=$nodelist
+                            NODE=\${NODELIST%%,*}
+                            trap 'set +e; set -x; ssh -i ci_key jenkins@\$NODE "set -ex; sudo umount \$IOF_BASE"' EXIT
+                            ssh -i ci_key jenkins@\$NODE "set -x
+                                set -e
+                                sudo mkdir -p \$IOF_BASE
+                                sudo mount -t nfs \$HOSTNAME:\$PWD \$IOF_BASE
+                                cd \$IOF_BASE
+                                ln -s /usr/bin/fusermount install/bin/fusermount3
+                                pip3.4 install --user tabulate
```

Comment:
I think you need to add a line here which says
IOF_TESTLOG=test/output/master-alloc_fail/

It looks like there might be a bug in iof_test_alloc_fail.py on L209, it should set a default value for IOF_TESTLOG, and then append the bin_cmd directory afterwards, otherwise it'll use the same log_dif for both the cnss and ionss tests, and the results will overwrite itself.

If you make the same logging changes to the existing Fault-injecction stage at the same time you'll be able to verify they're working as part of this PR.